tekton/pipeline_run.go

```diff
@@ -163,6 +163,21 @@ func (r *ReleasePipelineRun) WithServiceAccount(serviceAccount string) *ReleaseP
 	return r
 }
 
+// WithTaskGitRevisionParameter adds the taskGitRevision parameter to the managed Release PipelineRun with the value of the revision
+// from the pipelineRef.
+func (r *ReleasePipelineRun) WithTaskGitRevisionParameter(pipelineRef *tektonv1.PipelineRef) *ReleasePipelineRun {
```

Comment:
Shouldn't we also check that the resolvername is "git" first? I understand that it's unlikely that there will be a "revision" used in another resolver (since we don't really use that many resolver types), but it seems like the right thing to do.

Also, what's the benefit of not working with our PipelineRef directly and calling `ToTektonPipelineRef` first? Is it for consistency with the other function? Any other reason?

Reply:
Totally. Updated to add a check for `git` resolver first

It was just for consistency with the other function, but I changed it to use the PipelineRef directly

Reply:
Cool, thanks.